### PR TITLE
Clarify how MRC works in FAQ

### PR DESCRIPTION
**Pull request by makeasnek**
Make MRC fee portion more readable, provide examples

### CODE REVIEW

File: wiki/faq.md [hunk 1]
```diff
@@ -270,23 +270,33 @@ for doing so is much higher than if you were to wait for longer.
...
 a transaction over the Gridcoin blockchain and is usually insignificant. This
 fee is charged when you make the MRC request.
 
 - MRC fee: This is the fee that is charged after the MRC request is processed
-and is applied to the rewards that you have claimed. The fee starts at 40% of
-the owed GRC at the 14-day interval since the last claim (or beacon creation)
-and decreases by `40% * 14/(number of days since last claim)`.
+and is applied to the rewards that you have claimed. The fee is equivalent to 
+roughly 5.6 days of crunching. If you request every two weeks (14 days), 5.6
+days is 40% of that. If you request every six weeks, it is 13%.
```
Comment (RoboticMind) [inline wiki/faq.md:287]: I'm not sure if the roughly 5.6 days of crunching is correct, but it's been a bit since I've looked at the formula MRC uses

Comment (makeasnek) [reply wiki/faq.md:287]: image

**Comment by barton2526** [comment]
This branch has a conflict now

**Comment by makeasnek** [comment]
K I think I fixed the conflict, this is my first time doing a conflict resolution so lmk if I did something wrong

**Comment by makeasnek** [comment]
Poke

**Comment by barton2526** [comment]
Please update the title to be more descriptive

**Comment by makeasnek** [comment]
done

**Review by barton2526 — APPROVED** [review]

**Pull request merged**